### PR TITLE
Mhr Corrections: Review Confirm

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#19507

*Description of changes:*
* Apply Correction Badge connections on Review Components for MHR Corrections
* Refactor/Consolidate correction models into composable for re-use amongst multiple UI instances. 
(Also for future filing-wide comparisons ie validations/submissions)
* Update/added unit testing
* Re-order Submitting Party/DocId on ReviewConfirm

Screenshot 2024-03-21 at 10 58 00 AM
________________________________________________________________________________________________________
Screenshot 2024-03-21 at 10 58 15 AM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1793-x646g16v.web.app

File: ppr-ui/src/composables/mhrRegistration/useMhrCorrections.ts [hunk 1]
```diff
@@ -44,6 +46,85 @@ export const useMhrCorrections = () => {
       ])
   })
 
+  /** Correction State Models: Used in multiple ui-locations for CORRECTED LABELS, centralized for re-use **/
```
Comment (dimak1) [inline ppr-ui/src/composables/mhrRegistration/useMhrCorrections.ts:49]: Nice, good idea!

**Review by doug-lovett — APPROVED** [review]
Looks great.

File: ppr-ui/tests/unit/MhrCorrections.spec.ts [hunk 2]
```diff
@@ -152,4 +154,59 @@ describe('Mhr Corrections', async () => {
     expect(homeSectionsBadges[1].text()).toContain(ActionTypes.ADDED)
   })
 
+  it('Home Location step: renders corrected badges', async () => {
+    goToStep(4)
+    // Since we are mounting from parent view, isolate selector to specific step to prevent Review Confirm Badge Counts
```
Comment (dimak1) [inline ppr-ui/tests/unit/MhrCorrections.spec.ts:159]: 👍🏻

**Review by dimak1 — APPROVED** [review]
Looks great!

**Pull request merged**